{"name": "test_verify_error_notification_without_credentials", "status": "passed", "start": 1709473028154, "stop": 1709473028346, "uuid": "31017c97-fad7-4aff-b159-32755e6702f0", "historyId": "ea42a91ea4c26ad2d2ef1729eba9a481", "testCaseId": "ea42a91ea4c26ad2d2ef1729eba9a481", "fullName": "Tests.test_login.TestSearch#test_verify_error_notification_without_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestSearch"}, {"name": "host", "value": "DESKTOP-UEVTOM8"}, {"name": "thread", "value": "3304-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login"}]}